### PR TITLE
More typo/spelling fixes

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
Only typo/spelling, no code.
I wasn't expecting to find so much typo.

### CODE REVIEW

File: include/openssl/ec.h [hunk 1]
```diff
@@ -452,7 +452,7 @@ typedef struct {
 
 /*
  * EC_builtin_curves(EC_builtin_curve *r, size_t size) returns number of all
- * available curves or zero if a error occurred. In case r ist not zero
+ * available curves or zero if a error occurred. In case r isn't not zero
```
Comment (richsalz) [inline include/openssl/ec.h]: "isn't not" ? Maybe "When r is not zero, then" ?

File: include/openssl/srp.h [hunk 2]
```diff
@@ -110,7 +110,7 @@ typedef struct SRP_VBASE_st {
 } SRP_VBASE;
 
 /*
- * Structure interne pour retenir les couples N et g
+ * Internal structure to retain the N and g couples
```
Comment (richsalz) [inline include/openssl/srp.h]: pairs not couples?

File: ssl/t1_lib.c [hunk 3]
```diff
@@ -2734,7 +2734,7 @@ int tls1_set_server_sigalgs(SSL *s)
 {
     int al;
     size_t i;
-    /* Clear any shared sigtnature algorithms */
+    /* Clear any shared signature algorithms */
```
Comment (richsalz) [inline ssl/t1_lib.c]: as long as you're hear, blank line after the local vars?

File: include/openssl/srp.h (hunk 2 above)
Comment (jrchamp) [inline include/openssl/srp.h]: Perhaps "contain"?

File: ssl/s3_enc.c [hunk 4]
```diff
@@ -231,7 +231,7 @@ int ssl3_change_cipher_state(SSL *s, int which)
             goto err;
         else
             /*
-             * make sure it's intialized in case we exit later with an error
+             * make sure it's initialised in case we exit later with an error
```
Comment (jrchamp) [inline ssl/s3_enc.c]: Need to pick either "s" or "z" and be consistent. Earlier you have serialized and serializing.

Comment (richsalz) [inline ssl/s3_enc.c]: > Need to pick either "s" or "z" and be consistent. Earlier you have serialized and serializing.

Style is to use English not American.

Comment (jrchamp) [inline ssl/s3_enc.c]: Ok, then this one is good. I'll drop a note on the serializ*

File: include/openssl/ct.h [hunk 5]
```diff
@@ -368,7 +368,7 @@ __owur int SCT_LIST_validate(const STACK_OF(SCT) *scts,
  * for data that caller is responsible for freeing (only if function returns
  * successfully).
  * If "pp" is NULL and "*pp" is not NULL, caller is responsible for ensuring
- * that "*pp" is large enough to accept all of the serializied data.
+ * that "*pp" is large enough to accept all of the serialized data.
```
Comment (jrchamp) [inline include/openssl/ct.h]: serialised

File: include/openssl/ct.h [hunk 6]
```diff
@@ -395,7 +395,7 @@ STACK_OF(SCT) *o2i_SCT_LIST(STACK_OF(SCT) **a, const unsigned char **pp,
...
```
Comment (jrchamp) [inline include/openssl/ct.h]: serialised

File: ssl/ssl_lib.c [hunk 7]
```diff
@@ -3712,7 +3712,7 @@ void SSL_set_not_resumable_session_callback(SSL *ssl,
 
 /*
  * Allocates new EVP_MD_CTX and sets pointer to it into given pointer
- * vairable, freeing EVP_MD_CTX previously stored in that variable, if any.
+ * variable, freeing EVP_MD_CTX previously stored in that variable, if any.
  * If EVP_MD pointer is passed, initializes ctx with this md Returns newly
```
Comment (jrchamp) [inline ssl/ssl_lib.c]: initialises

File: ssl/s3_enc.c (hunk 4 above)
Comment (jrchamp) [inline ssl/s3_enc.c]: To clarify: British English or Oxford English? https://en.wikipedia.org/wiki/Oxford_spelling

Comment (richsalz) [inline ssl/s3_enc.c]: A level of granularity beyond my ability to discern.

Comment (jrchamp) [inline ssl/s3_enc.c]: It appears that the OED prefers -ize

File: ssl/ssl_lib.c (hunk 7 above)
Comment (FdaSilvaYY) [inline ssl/ssl_lib.c]: I don't need to put a finger in this.. ;)
A grep on codebase show that it's nearly a draw ( match nul ...).

Comment (jrchamp) [inline ssl/ssl_lib.c]: Given that the coding style does not specify the specific spelling style to use (and that typically both are acceptable in British and Oxford English), I'm fine with that. Just wanted to note it when my search found it in a block being modified.

**Comment by richsalz** [comment]
+1

File: ssl/ssl_err.c [hunk 8]
```diff
@@ -588,7 +588,7 @@ static ERR_STRING_DATA SSL_str_reasons[] = {
     {ERR_REASON(SSL_R_RENEGOTIATION_MISMATCH), "renegotiation mismatch"},
     {ERR_REASON(SSL_R_REQUIRED_CIPHER_MISSING), "required cipher missing"},
     {ERR_REASON(SSL_R_REQUIRED_COMPRESSSION_ALGORITHM_MISSING),
-     "required compresssion algorithm missing"},
+     "required compression algorithm missing"},
```
Comment (kroeckx) [inline ssl/ssl_err.c]: Fix the error code itself too?

Comment (FdaSilvaYY) [inline ssl/ssl_err.c]: Fixed in a separated commit.

**Comment by FdaSilvaYY** [comment]
Merged as 620d540bd47a
So closing.

**Comment by t-j-h** [comment]
+1

**Pull request closed without merging**